docs/integration/webapi.rst

```diff
@@ -228,6 +228,31 @@ Autofac introduces the new interfaces to allow you to concentrate on implementin
 
 Another reason for creating the internal attribute wrappers is to support the ``InstancePerRequest`` lifetime scope for filters. See below for more on that.
 
+Setting the Response in an Autofac Action Filter
+------------------------------------------------
+
+In the same way as with standard Web API filters,  you are able to set the HttpResponse object in the
+``OnActionExecutingAsync`` method of an action filter.
+
+.. sourcecode:: csharp
+
+  class RequestRejectionFilter : IAutofacActionFilter
+  {
+    public Task OnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
+    {
+      // Request is not valid for some reason.
+      actionContext.Response = actionContext.Request.CreateErrorResponse(HttpStatusCode.BadRequest, "Request not valid");
+      return Task.FromResult(0);
+    }
+
+    public void Task OnActionExecutedAsync(HttpActionExecutedContext actionExecutedContext, CancellationToken cancellationToken)
+    {
+    }
+  }
+
+To match the standard Web API behaviour, if you set the Response object, then no subsequent action filters will be 
+invoked. However, any action filters already invoked will have OnActionExecutedAsync called with the appropriate response populated.
```

Comment:
Don't forget the double-backticks to make `OnActionExecutedAsync` and `HttpResponseMessage` show up as keywords.

Reply:
Changed 'object' to 'property' to make it clear I mean the property on the context. Added backticks.